content/en/docs/ops/ambient/usage/traffic-redirection/index.md

```diff
@@ -0,0 +1,134 @@
+
+{{< text bash >}}
+$ kubectl debug $(kubectl get pod -l app=sleep -n ambient-demo -o jsonpath='{.items[0].metadata.name}') -it -n ambient-demo  --image nicolaka/netshoot  -- ss -ntlp
+Defaulting debug container name to debugger-nhd4d.
+State  Recv-Q Send-Q Local Address:Port  Peer Address:PortProcess
+LISTEN 0      128        127.0.0.1:15080      0.0.0.0:*
+LISTEN 0      128                *:15006            *:*
+LISTEN 0      128                *:15001            *:*
+LISTEN 0      128                *:15008            *:*
+{{< /text >}}
+
+### Check the iptables rules setup
+
+To view the iptables rules setup inside one of the application pods, execute this command:
+
+{{< text bash >}}
+$ kubectl debug $(kubectl get pod -l app=sleep -n ambient-demo -o jsonpath='{.items[0].metadata.name}') -it --image gcr.io/istio-release/base --profile=netadmin -n ambient-demo -- iptables-save
+
+Defaulting debug container name to debugger-m44qc.
+# Generated by iptables-save v1.8.7 on Wed Feb 21 20:38:16 2024
+*mangle
+:PREROUTING ACCEPT [320:53261]
+:INPUT ACCEPT [23753:267657744]
+:FORWARD ACCEPT [0:0]
+:OUTPUT ACCEPT [23352:134432712]
+:POSTROUTING ACCEPT [23352:134432712]
+:ISTIO_OUTPUT - [0:0]
+:ISTIO_PRERT - [0:0]
+-A PREROUTING -j ISTIO_PRERT
+-A OUTPUT -j ISTIO_OUTPUT
+-A ISTIO_OUTPUT -m connmark --mark 0x111/0xfff -j CONNMARK --restore-mark --nfmask 0xffffffff --ctmask 0xffffffff
+-A ISTIO_PRERT -m mark --mark 0x539/0xfff -j CONNMARK --set-xmark 0x111/0xfff
+-A ISTIO_PRERT -s 169.254.7.127/32 -p tcp -m tcp -j ACCEPT
+-A ISTIO_PRERT ! -d 127.0.0.1/32 -i lo -p tcp -j ACCEPT
+-A ISTIO_PRERT -p tcp -m tcp --dport 15008 -m mark ! --mark 0x539/0xfff -j TPROXY --on-port 15008 --on-ip 0.0.0.0 --tproxy-mark 0x111/0xfff
+-A ISTIO_PRERT -p tcp -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
+-A ISTIO_PRERT ! -d 127.0.0.1/32 -p tcp -m mark ! --mark 0x539/0xfff -j TPROXY --on-port 15006 --on-ip 0.0.0.0 --tproxy-mark 0x111/0xfff
+COMMIT
+# Completed on Wed Feb 21 20:38:16 2024
+# Generated by iptables-save v1.8.7 on Wed Feb 21 20:38:16 2024
+*nat
+:PREROUTING ACCEPT [0:0]
+:INPUT ACCEPT [0:0]
+:OUTPUT ACCEPT [175:13694]
+:POSTROUTING ACCEPT [205:15494]
+:ISTIO_OUTPUT - [0:0]
+-A OUTPUT -j ISTIO_OUTPUT
+-A ISTIO_OUTPUT -d 169.254.7.127/32 -p tcp -m tcp -j ACCEPT
+-A ISTIO_OUTPUT -p tcp -m mark --mark 0x111/0xfff -j ACCEPT
+-A ISTIO_OUTPUT ! -d 127.0.0.1/32 -o lo -j ACCEPT
+-A ISTIO_OUTPUT ! -d 127.0.0.1/32 -p tcp -m mark ! --mark 0x539/0xfff -j REDIRECT --to-ports 15001
+COMMIT
+{{< /text >}}
+
+The command output shows that additional Istio-specific chains are added to the NAT and Mangle tables in netfilter/iptables within the application pod's network namespace. Traffic coming into the pod on TCP port 15008 or 15006 is redirected into the ztunnel proxy for ingress processing. The traffic leaving the pod is redirected to ztunnel's port 15001 for egress processing, before being sent out by ztunnel using HBONE encapsulation.
```

Comment:
`15008 or 15006` - Is it just inbound traffic on these two ports are redirected to ztunnel? I had thought it is all traffic (excluding healthcheck perhaps?)

cc @bleggett

Reply:
Yep - clarified a bit.